### PR TITLE
Update SDK landing page with new TOCs

### PR DESCRIPTION
**Pull request by cbush**
Adopted by @mongomoe 

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/

## Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14742

## Docs staging link (requires sign-in on MongoDB Corp SSO):
SDK landing page: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/
iOS fundamentals.txt: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/ios/fundamentals/
iOS data-types.txt: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/ios/data-types/
iOS examples.txt: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/ios/examples/
iOS advanced-guides.txt: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/sdk/ios/advanced-guides/
React Native database.txt: https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/sdk-landing/react-native/database/

### CODE REVIEW

File: source/sdk.txt [hunk 1]
```diff
@@ -27,37 +27,75 @@ Android SDK
...
-.. TODO
-.. iOS/macOS SDK
-.. ~~~~~~~~~~~~~
+iOS, macOS, tvOS, and watchOS SDK
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :doc:`Install Realm for iOS, macOS, tvOS, and watchOS </sdk/ios/install>`
+- :doc:`Quick Start with Sync </sdk/ios/quick-start-with-sync>`
+- :doc:`SwiftUI & Combine </sdk/ios/swiftui>`
+- :doc:`Fundamentals </sdk/ios/fundamentals>`
```
Comment (mongomoe) [inline source/sdk.txt:47]: The IOS fundamentals, usage examples and data types pages don't have any content in them, is this okay?

Comment (mongomoe) [reply source/sdk.txt:47]: whereas the android pages do

File: source/sdk.txt [hunk 2]
```diff
@@ -27,37 +27,75 @@ Android SDK
...
 
 React Native SDK
 ~~~~~~~~~~~~~~~~
 
 - :doc:`Install Realm for React Native </react-native/install>`
-- :doc:`React Native Quick Start </react-native/quick-start>`
-- :doc:`React Native  </react-native/quick-start>`
+- :doc:`Quick Start </react-native/quick-start>`
+- :doc:`The Realm Data Model </react-native/data-model>`
+- :doc:`Realm Database </react-native/database>`
```
Comment (mongomoe) [inline source/sdk.txt:75]: The React Native > Realm Database page looks like it has no content on it either

**Review by mongomoe — APPROVED** [review]
LGTM w/ a couple comments

File: source/sdk.txt (hunk 2 above)
Comment (mongomoe) [reply source/sdk.txt:75]: update: create a [ticket](https://jira.mongodb.org/browse/DOCSP-14742) for this work

File: source/sdk.txt (hunk 1 above)
Comment (mongomoe) [reply source/sdk.txt:47]: update: create a [ticket](https://jira.mongodb.org/browse/DOCSP-14742) for this work

File: source/sdk.txt (hunk 2 above)
Comment (nlarew) [reply source/sdk.txt:75]: This page should go away once we port RN to use the new IA format (same with the RN **Realm Database** page)

**Review by nlarew — APPROVED** [review]

File: source/sdk.txt (hunk 2 above)
Comment (mongomoe) [reply source/sdk.txt:75]: fair point, I decided to add this content anyway (as part of this PR: https://github.com/cbush/docs-realm/pull/2). It felt a little ugly to keep an empty page, even if it's going to go away in a few days. Luckily it only took a few minutes to do.

**Pull request merged**